pinot-spi/src/main/java/org/apache/pinot/spi/config/table/QuotaConfig.java

```diff
@@ -35,7 +35,8 @@ public class QuotaConfig extends BaseJsonConfig {
   private static final long INVALID_STORAGE_IN_BYTES = -1L;
   private static final double INVALID_MAX_QPS = -1.0;
 
-  @JsonPropertyDescription("Storage allocated for this table, e.g. \"10G\"")
+
+  @JsonPropertyDescription("Storage allocated for this table/database, e.g. \"10G\"")
```

Comment:
Is storage quota implemented ?

Reply:
It’s not yet. But planning to do that as well soon

Reply:
In the same PR ? If not, I suggest removing it.